src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -155,16 +156,16 @@ public void perform(
                 run.setResult(Result.FAILURE);
             }
         } catch (ExecutionException e) {
-            outputStream.println("There was an execution error trying to run your journeys in mabl");
+            outputStream.println("There was an execution error trying to run your tests in mabl");
             e.printStackTrace(outputStream);
             if(continueOnMablError) {
                 run.setResult(Result.FAILURE);
             } else {
                 run.setResult(Result.SUCCESS);
             }
         } catch (TimeoutException e) {
-            outputStream.printf("Oh dear. Your journeys exceeded the max plugin runtime limit of %d seconds.%n" +
-                    "We've aborted this Jenkins step, but your journeys may still be running in mabl.", EXECUTION_TIMEOUT_SECONDS);
+            outputStream.printf("Oh dear. Your tests exceeded the max plugin runtime limit of %d seconds.%n" +
```

Comment:
Not really part of your PR, but I'm not sure I like the conversational tone of "Oh dear." here, but that might just be me.

Reply:
Probably good to temper the voice here. We're no longer trying to be a javial, personified machine TTBOMK.

Reply:
Let me work on this on a follow-up PR.